ArduCopter/events.cpp

```diff
@@ -306,6 +306,77 @@ void Copter::gpsglitch_check()
     }
 }
 
+// executes lean-angle failsafe if lean angle is too high for longer than a set threshold
+// indicates the wind is too high to fly safely in planck mode
+void Copter::failsafe_lean_check(uint16_t duration_ms)
```

Comment:
Where is `duration_ms` used?

Reply:
I think a recent change didn't get pushed, I will fix